control/cli.py

```diff
@@ -99,23 +99,23 @@ def logger(self):
             raise AttributeError("logger is None. Set with connect method.")
         return self._logger
 
-    def connect(self, nvme_config):
+    def connect(self, settings):
```

Comment:
I see some places I can more consistently call it "settings" (specifically in __main__.py & settings.py). My goal was to separate the naming in reference to the user settings in the gateway "config" file from the persistent "config". I don't know that I could be more consistent in cli.py without ceasing to refer to the config file as a "config" file. Do you mean you think it should no longer be called "main.config" and we should refer to it as a "settings" file?

Reply:
Yes either we call everything related to the local config file settings or config but I think the mix is confusing. I think changing it to `settings` everywhere makes sense to avoid confusion with the OMAP/in-memory config.